docs/submitting_backend.md

```diff
@@ -0,0 +1,205 @@
+Thanks for being interested in contributing! All contributions should be submitted as pull requests
+to the [Wagtail Content Import repository](https://github.com/torchbox/wagtail-content-import).
+
+
+## Submitting a New Picker
+
+If you're planning on submitting a new picker - apps which enable choosing and importing a file from a 
+remote source - you'll need to follow this blueprint:
+
+###Overview and File Structure
```

Comment:
I think you need a space between ### and title (here and for subtitles below)

Reply:
Added now, thanks